src/client/src/apps/MainRoute/store/applicationServices.ts

```diff
@@ -13,19 +13,23 @@ import {
 import { User } from 'rt-types'
 import { ReplaySubject } from 'rxjs'
 import { retryWhen, multicast, refCount } from 'rxjs/operators'
-import { OpenFinLimitChecker } from './shell/openFin'
-import { referenceDataService } from './shell/referenceData'
+import { referenceDataService } from '../data/referenceData'
+import { LimitChecker } from 'rt-platforms/platformAdapter'
```

Comment:
`import { LimitChecker } from 'rt-platforms'`

expose the Limit checker interface through an index.ts